SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2836,6 +2846,11 @@ void USpatialNetDriver::QueryRoutingPartition()
 								NetDriver->RoutingWorkerId = Entity.entity_id;
 								NetDriver->QueryRoutingPartition();
 							}
+							else if (WorkerType == SpatialConstants::StrategyWorkerType.ToString())
+							{
+								NetDriver->StrategyWorkerId = Entity.entity_id;
```

Comment:
We might not need that going forward, I expect the routing and strategy partition to be statically assigned when creating the initial snapshot

Reply:
Ok, I have removed it for now.